README.md

```diff
@@ -28,18 +28,18 @@
-| CQL_SERVER_BASE_URL          | Url of the local FHIR server that handles CQL requests                                                                                                                 |                  | http://cql                                                           |
-| API_BASE_URL                 | sets the base url of the webservice, this is necessary if the webservice is running behind a proxy server, if not filled the api base url is the request url           | https://host/api |                                                                      |
+| FLARE_WEBSERVICE_BASE_URL    | URL of the local FLARE webservice - needed for FHIR query translation and when running the DIRECT path                                                                 |                  | http://localhost:5000                                                |
+| CQL_SERVER_BASE_URL          | URL of the local FHIR server that handles CQL requests                                                                                                                 |                  | http://cql                                                           |
+| API_BASE_URL                 | Sets the base URL of the webservice. This is necessary if the webservice is running behind a proxy server. If not filled, the API base URL is the request URL          | https://host/api |                                                                      |
 | QUERY_VALIDATION_ENABLED     | When enabled, any structured query submitted via the `run-query` endpoint is validated against the JSON schema located in `src/main/resources/query/query-schema.json` | true / false     | true                                                                 |
 | QUERYRESULT_EXPIRY_MINUTES   | How many minutes should query results be kept in memory?                                                                                                               |                  | 5                                                                    |
-| QUERYRESULT_PUBLIC_KEY       | The public key in Base64-encoded DER format without banners and line breaks.                                                                                           |                  |                                                                      |
+| QUERYRESULT_PUBLIC_KEY       | The public key in Base64-encoded DER format without banners and line breaks.                                                                                           |                                                                           
 | ALLOWED_ORIGINS              | Allowed origins for cross-origin requests. This should at least cover the frontend address.                                                                            |                  | http://localhost                                                     |
 
```

Comment:
@michael-82  some new env vars are missing - please update

Reply:
I compared it with the [list in the deployment repo](https://github.com/medizininformatik-initiative/feasibility-deploy/blob/main/feasibility-portal/README.md) and did not find anything missing. Please let me know what I am missing